travvy/src/components/AttractionsList.vue

```diff
@@ -118,13 +105,82 @@
         </gmap-map>
       </div>
       <div v-else-if="$store.state.city == 'Rome'">
-        <gmap-map v-bind:center="{lat: 41.9028, lng: 12.4964}" v-bind:zoom="11" style="height: 425px">
+        <gmap-map v-bind:center="{lat: 41.9028, lng: 12.4964}" v-bind:zoom="11" style="height: 725px">
         </gmap-map>
       </div>
       <div v-else="">
       </div>
       </div>
       <br>
+      <br>
+      <br>
+      <hr size="3" width="60%" color="#E7EDF7"> 
+
+<!-- all reviews taken from TripAdvisor-->
+<!-- the reviews of each city will dynamically display based on the chosen destination -->
+      <h1>Traveller Tips</h1>
```

Comment:
These all look great Lucas. Do you think we should say something like "Reviews from Trip Advisor"?

Reply:
Just have it commented but I dont think its necessary